src/NLog/Targets/FileTarget.cs

```diff
@@ -201,8 +200,17 @@ public FileTarget()
             this.previousFileNames = new Queue<string>(this.maxLogFilenames);
             this.fileAppenderCache = FileAppenderCache.Empty;
             this.CleanupFileName = true;
+            
+            this.Compressor = DefaultCompressor;
```

Comment:
I'm a bit confused by the Compressor  and DefaultCompressor properties. Are both needed? Is this because of the `internal` implementation?